sites/partners/src/components/listings/ListingFormActions.tsx

```diff
@@ -364,7 +380,8 @@ const ListingFormActions = ({
             elements.push(editFromDetailButton)
         }
 
-        // all users can preview
+        // all users can copy or preview
+        elements.push(copyButton)
```

Comment:
This should only be visible for everyone on Doorway, HBA and Core should only have this allowed for Admin